lib/smart_answer_flows/calculate-statutory-sick-pay.rb

```diff
@@ -302,48 +302,36 @@ def define
       end
 
       # Answer 1
-      outcome :already_getting_maternity
+      outcome :already_getting_maternity, use_outcome_templates: true
```

Comment:
Yes! I completely overlooked that final step of using `Flow#use_outcome_templates`. Thanks @tadast.